app/graphql/mutations/comment_survey.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: false
+
+module Mutations
+  class CommentSurvey < BaseMutation
+    argument :survey_id, ID, required: true
+    argument :message, String, required: true
+
+    type Types::StatusType
+
+    def resolve(survey_id:, message:)
+      survey = Survey::Poll.find_by(id: survey_id)
+
+      return error_status("Access not permitted: Survey missing") if survey.blank?
+      return error_status("Access not permitted: Survey already archived") if survey.archived?
+
+      comment = survey.comments.build(message: message)
+      return error_status("Access not permitted: Comment invalid, #{comment.error_messages}") unless comment.valid?
+
+      begin
+        comment.save
+        OpenStruct.new(id: survey_id, status: "comment successfully", status_code: 200)
```

Comment:
hier sollte die id des kommentars zurück und nicht die des surveys, oder die zusätzlich. aber id müsste von dem erstellten datensatz sein. kümmere mich drum.

Reply:
anders, ich mach textlich ein `commented survey 21 successfully` draus

Reply:
![Uploading Bildschirmfoto 2021-07-06 um 17.37.18.png…]()

Reply:
done in 553f52f885d86e97136d5e4c9e0e8d1bd06afa6b